api/v1alpha1/lvmcluster_types.go

```diff
@@ -31,9 +31,9 @@ type LVMClusterSpec struct {
 	// Tolerations to apply to nodes to act on
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
-	// DeviceClasses are a rules that assign local storage devices to volumegroups that are used for creating lvm based PVs
+	// Storage describes the deviceClass configuration for local storage devices
```

Comment:
To keep it future-ready, should we update the description of the `Storage` field to be more generic or less aligned with `DeviceClass`; Thoughts?

Reply:
A good point. How about "Storage describes the configuration for local storage devices" ?

Reply:
Sounds Great!